README.md

```diff
@@ -22,26 +30,78 @@ browzine.script.src = "https://s3.amazonaws.com/browzine-adapters/summon/browzin
 document.head.appendChild(browzine.script);
 ```
 
+_BrowZine 360 Core Adapter Script_
+
+```
+var browzine = {
+  api: "https://api.thirdiron.com/public/v1/libraries/XXX",
+  apiKey: "ENTER API KEY",
+};
+
+browzine.script = document.createElement("script");
+browzine.script.src = "https://s3.amazonaws.com/browzine-adapters/360-core/browzine-360-core-adapter.js";
+document.head.appendChild(browzine.script);
+```
+
+_Loading Discovery Service Adapters_
+
+```
+var browzine = {
+  api: "https://api.thirdiron.com/public/v1/libraries/XXX",
+  apiKey: "ENTER API KEY",
+};
+
+browzine.script = document.createElement("script");
+browzine.script.src = "https://s3.amazonaws.com/browzine-adapters/summon/browzine-summon-adapter.js";
+document.head.appendChild(browzine.script);
+
+browzine.script = document.createElement("script");
+browzine.script.src = "https://s3.amazonaws.com/browzine-adapters/360-core/browzine-360-core-adapter.js";
+document.head.appendChild(browzine.script);
+```
+
+_Adding Summon Custom Script_
+
 Summon Custom Scripts are added in the Summon 2.0 External Script portion of the Summon Editor Settings:
 ![Summon 2.0 Editor Link](https://i.imgur.com/HJFpDGm.png "Summon 2.0 Editor Link")
 
 As a Custom Script Url:
 ![Summon 2.0 External Script](https://i.imgur.com/piLMSic.png "Summon 2.0 External Script")
 
+
+_Adding 360 Core Custom Script_
+
+360 Core custom scripts are added in 360 Core > Administration Console > E-Journal Portal 2.0 > Branding Options > Reference External JavaScript > JavaScript file url:
+![360 Core 2.0 External Script](https://i.imgur.com/c24Tlh5.png "360 Core 2.0 External Script")
+![360 Core 2.0 External Script](https://i.imgur.com/l53rUL8.png "360 Core 2.0 External Script")
+
+
 ## Customize The BrowZine Enhancement
 
 Customize the naming conventions for each type of search result - Journal/Article - by changing the wording in the quotes below:
 
 E.g. You can customize "View the Journal" and "View Complete Issue" or "Browse Now". These customizations are optional and the defaults are shown below.
 
+_Summon Customizations_
+
 ```
 var browzine = {
   api: "https://api.thirdiron.com/public/v1/libraries/XXX",
   apiKey: "ENTER API KEY",
-  journalWording: "View the Journal",
-  articleWording: "View Complete Issue",
-  journalBrowZineWebLinkText: "Browse Now",
-  articleBrowZineWebLinkText: "Browse Now",
+  summonJournalWording: "View the Journal",
```

Comment:
Definitely cleaner naming for folks just dropping this into their systems I think 👍